docs/installation-and-operations/configuration/environment/README.md

```diff
@@ -142,7 +142,7 @@ OPENPROJECT_AUTOLOGIN (default=0) Autologin
 OPENPROJECT_AUTOLOGIN__COOKIE__NAME (default="autologin") Cookie name for autologin cookie
 OPENPROJECT_AUTOLOGIN__COOKIE__PATH (default="/") Cookie path for autologin cookie
 OPENPROJECT_AVAILABLE__LANGUAGES (default=["ca", "cs", "de", "el", "en", "es", "fr", "hu", "id", "it", "ja", "ko", "lt", "nl", "no", "pl", "pt-BR", "pt-PT", "ro", "ru", "sk", "sl", "sv", "tr", "uk", "vi", "zh-CN", "zh-TW"]) Available languages
-OPENPROJECT_AVATAR__LINK__EXPIRY__SECONDS (default=86400) Cache duration for avatar image API responses
+OPENPROJECT_AVATAR__LINK__EXPIRATION__SECONDS (default=86400) Cache duration for avatar image API responses
```

Comment:
@oliverguenther Not sure of the best way to handle renaming environment variables in deployments?